docker/Dockerfile

```diff
@@ -15,7 +15,7 @@ FROM ethereum/solc:0.6.4-alpine as bsc-genesis
 RUN apk add --d --no-cache ca-certificates npm nodejs bash alpine-sdk
 
 RUN git clone https://github.com/binance-chain/bsc-genesis-contract.git /root/genesis \
-    && rm /root/genesis/package-lock.json && cd /root/genesis && npm install
```

Comment:
better check with the author why `rm the package-lock.json`?

Reply:
checked, but j76589 forget the reason
after discussion with him, we decide to keep package-lock.json